Build failed on Mac

Running a `make dist` on mac from a clean clone failed.

```
mkdir -p .build/externs/
wget -O .build/externs/angular-1.6-q_templated.js https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= github_versions | cut --delimiter = --fields 2`/contrib/externs/angular-1.6-q_templated.js
cut: illegal option -- -
usage: cut -b list [-n] [file ...]
       cut -c list [file ...]
       cut -f list [-s] [-d delim] [file ...]
--2017-05-15 11:37:01--  https://raw.githubusercontent.com/google/closure-compiler//contrib/externs/angular-1.6-q_templated.js
Resolving raw.githubusercontent.com... 151.101.36.133
Connecting to raw.githubusercontent.com|151.101.36.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-05-15 11:37:02 ERROR 404: Not Found.

make: *** [.build/externs/angular-1.6-q_templated.js] Error 8
```
The closure version is not taken from the github-versions file.

OSX 10.12.4

Thx it fixes theissue but antoher is showing up:
```
npm install
npm ERR! git rev-list -n1 during_filter: fatal: ambiguous argument 'during_filter': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 during_filter: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 during_filter: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 during_filter: 
npm ERR! git rev-list -n1 during_filter: fatal: ambiguous argument 'during_filter': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 during_filter: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 during_filter: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 during_filter: 
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: Cloning into bare repository '/Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab'...
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: Warning: Permanently added the RSA host key for IP address '192.30.253.112' to the list of known hosts.
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: Permission denied (publickey).
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: fatal: Could not read from remote repository.
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: 
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: Please make sure you have the correct access rights
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: and the repository exists.
npm ERR! Darwin 16.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v6.10.0
npm ERR! npm  v3.10.10
npm ERR! code 128

npm ERR! Command failed: git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab
npm ERR! Cloning into bare repository '/Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab'...
npm ERR! Warning: Permanently added the RSA host key for IP address '192.30.253.112' to the list of known hosts.
npm ERR! Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.

```

--- a/Makefile
+++ b/Makefile
@@ -213,7 +213,7 @@ lint: .build/eslint.timestamp
 
 .PHONY: git-attributes
 git-attributes:
-	git --no-pager diff --check `git log --oneline | tail -1 | cut --fields=1 --delimiter=' '`
+	git --no-pager diff --check `git log --oneline | tail -1 | cut -f 1 -d ' '`
 
 .PHONY: eof-newline
 eof-newline:
@@ -790,26 +790,26 @@ contribs/gmf/fonts/fontawesome-webfont.%: node_modules/font-awesome/fonts/fontaw
 
 contribs/gmf/build/angular-locale_%.js: github_versions
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/angular/angular.js/`grep ^angular.js= $< | cut --delimiter = --fields 2`/src/ngLocale/angular-locale_$*.js
+	wget -O $@ https://raw.githubusercontent.com/angular/angular.js/`grep ^angular.js= $< | cut -d = -f 2`/src/ngLocale/angular-locale_$*.js
 
 $(EXTERNS_ANGULAR): github_versions
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut --delimiter = --fields 2`/contrib/externs/angular-1.6.js
+	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut -d = -f 2`/contrib/externs/angular-1.6.js
 	touch $@
 
 $(EXTERNS_ANGULAR_Q): github_versions
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut --delimiter = --fields 2`/contrib/externs/angular-1.6-q_templated.js
+	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut -d = -f 2`/contrib/externs/angular-1.6-q_templated.js
 	touch $@
 
 $(EXTERNS_ANGULAR_HTTP_PROMISE): github_versions
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut --delimiter = --fields 2`/contrib/externs/angular-1.6-http-promise_templated.js
+	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut -d = -f 2`/contrib/externs/angular-1.6-http-promise_templated.js
 	touch $@
 
 $(EXTERNS_JQUERY): github_versions
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut --delimiter = --fields 2`/contrib/externs/jquery-1.9.js
+	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut -d = -f 2`/contrib/externs/jquery-1.9.js
 	touch $@
 
 .build/python-venv:
@@ -829,7 +829,7 @@ $(EXTERNS_JQUERY): github_versions
 .build/closure-library: github_versions
 	mkdir -p $(dir $@)
 	git clone http://github.com/google/closure-library/ $@
-	cd $@; git checkout `grep ^closure-library= $< | cut --delimiter = --fields 2`
+	cd $@; git checkout `grep ^closure-library= $< | cut -d = -f 2`
 
 .build/ol-deps.js: .build/python-venv .build/node_modules.timestamp
 	.build/python-venv/bin/python buildtools/closure/depswriter.py \

--- a/Makefile_windows
+++ b/Makefile_windows
@@ -160,7 +160,7 @@ lint: .build/eslint.timestamp
 
 .PHONY: git-attributes
 git-attributes:
-	git --no-pager diff --check `git log --oneline | tail -1 | cut --fields=1 --delimiter=' '`
+	git --no-pager diff --check `git log --oneline | tail -1 | cut -f 1 -d ' '`
 
 .PHONY: test
 test: .build/ol-deps.js .build/ngeo-deps.js .build/gmf-deps.js .build/templatecache.js .build/gmftemplatecache.js .build/node_modules.timestamp
@@ -286,26 +286,26 @@ contribs/gmf/fonts/fontawesome-webfont.%: node_modules/font-awesome/fonts/fontaw
 
 contribs/gmf/build/angular-locale_%.js: github_versions
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/angular/angular.js/`grep ^angular.js= $< | cut --delimiter = --fields 2`/src/ngLocale/angular-locale_$*.js
+	wget -O $@ https://raw.githubusercontent.com/angular/angular.js/`grep ^angular.js= $< | cut -d = -f 2`/src/ngLocale/angular-locale_$*.js
 
 $(EXTERNS_ANGULAR): github_versions
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut --delimiter = --fields 2`/contrib/externs/angular-1.6.js
+	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut -d = -f 2`/contrib/externs/angular-1.6.js
 	touch $@
 
 $(EXTERNS_ANGULAR_Q): github_versions
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut --delimiter = --fields 2`/contrib/externs/angular-1.6-q_templated.js
+	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut -d = -f 2`/contrib/externs/angular-1.6-q_templated.js
 	touch $@
 
 $(EXTERNS_ANGULAR_HTTP_PROMISE): github_versions
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut --delimiter = --fields 2`/contrib/externs/angular-1.6-http-promise_templated.js
+	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut -d = -f 2`/contrib/externs/angular-1.6-http-promise_templated.js
 	touch $@
 
 $(EXTERNS_JQUERY): github_versions
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut --delimiter = --fields 2`/contrib/externs/jquery-1.9.js
+	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= $< | cut -d = -f 2`/contrib/externs/jquery-1.9.js
 	touch $@
 
 .build/python-venv:
@@ -323,7 +323,7 @@ $(EXTERNS_JQUERY): github_versions
 .build/closure-library: github_versions
 	mkdir -p $(dir $@)
 	git clone http://github.com/google/closure-library/ $@
-	cd $@; git checkout `grep ^closure-library= $< | cut --delimiter = --fields 2`
+	cd $@; git checkout `grep ^closure-library= $< | cut -d = -f 2`
 
 .build/ol-deps.js: .build/python-venv .build/node_modules.timestamp
 	.build/python-venv/Scripts/python buildtools/closure/depswriter.py \

